plugins/events-backend-module-bitbucket-server/src/index.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2020 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * The module "bitbucket-server" for the Backstage backend plugin "events-backend"
+ * adding an event router for Bitbucket Server.
+ *
+ * @packageDocumentation
+ */
+
+export { BitbucketServerEventRouter } from './router/BitbucketServerEventRouter';
```

Comment:
export for the plugin module (`services/eventsModuleBitbucketServerEventRouter`) is missing. Also, it should be exported `as default`.

Reply:
@pjungermann I will fix this. Do you have any ideas as to what is causing so many failures in my checks? I rebased from a fresh main branch and all my changes were on the two plugins I was working on, but the running yarn tsc shows many more errors. I tested on my own BBS instance and all of my changes were working but I am confused as to why yarn tsc does not compile in my code with code that was written outside of my scope.

Reply:
Just did this in the latest commit, please let me know if I did it incorrectly.